lib/tests/tests/functions/test.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Can you rename this to something more descriptive? E.g. `parse_groks_alias_source.json`